stack.go

```diff
@@ -0,0 +1,21 @@
+package slogx
+
+import (
+	"fmt"
+	"log/slog"
+	"runtime"
+)
+
+func Stack() slog.Attr {
```

Comment:
One more thing: never add exported identifiers without documentation in a libraries like this one. Linter usually controls this, but it's temporary disabled, so just don't forget about this.